### ISSUE TITLE
Migration to React

### ISSUE THREAD

**Issue by supreme-gg-gg** [supreme-gg-gg/corpo-lingo#1]
Currently we do this with bootstrapping. Investigate how bootstrapping works, and implement and test a solution starting with `index.html`, then expanding it to `competition.html`.

**Comment by Anikkk** [comment]
HI @supreme-gg-gg i have one suggestion, we can make this a react project

**Comment by supreme-gg-gg** [comment]
> HI @supreme-gg-gg i have one suggestion, we can make this a react project

Yea I agree that was the original plan, we made this in a rush lol so we just used simple html and css.

**Comment by Anikkk** [comment]
I can help in making using react

**Comment by supreme-gg-gg** [comment]
That will be great, we are not currently working on front end anymore so feel free to experiment and submit a PR

**Comment by Anikkk** [comment]
Hey Chiang, I have made implemented the UI in react, i have made a separate react project i was not sure how to implement react project in the existing folder, so i made the separate folder "corpo-lingo-react" and made the UI for index.html and compete.html in react. functionality is yet to implement.

**Comment by supreme-gg-gg** [comment]
Yes, you can just raise a PR to main and I will have a look at what's there currently

**Comment by Anikkk** [comment]
ok sure

**Comment by Anikkk** [comment]
Hey @supreme-gg-gg  i have created the PR did you get the time to review it ?
i have created a new folder corpo-lingo-react, cd to that folder and then run the command npm start to run on the browser.

**Comment by supreme-gg-gg** [comment]
Sorry for the delay, I have looked at it and it is great! If you could go ahead to implement functionality I could do some more testing.

**Comment by Anikkk** [comment]
Np !! sure i am working on functionality

**Comment by supreme-gg-gg** [comment]
@Anikkk just want to check in on the status of this issue, I'm happy to take it over to keep things moving if you're not available right now

**Comment by Anikkk** [comment]
Hey @supreme-gg-gg  sry for the delay, currently my mid term exams are coming so not able to contribute, i will implement that but need some time.

**Comment by supreme-gg-gg** [comment]
No problem that's good, no rush and good luck with midterms

**Comment by Anikkk** [comment]
thanks man!!
